--- libtorrent-0.13/m4/ax_boost_filesystem.m4~	2007-11-28 03:23:16.000000000 +0000
+++ libtorrent-0.13/m4/ax_boost_filesystem.m4	2008-06-11 04:09:35.000000000 +0000
@@ -39,9 +39,9 @@
   fi])
   AC_LANG_SAVE
   AC_LANG_CPLUSPLUS
-  for ax_lib in $ax_filesystem_lib $ax_boost_filesystem_lib boost_filesystem-mt boost_filesystem; do
+  for ax_lib in $ax_filesystem_lib $ax_boost_filesystem_lib boost_filesystem; do
     ax_save_LIBS="$LIBS"
-    LIBS="$LIBS -l$ax_lib"
+    LIBS="$LIBS -l$ax_lib -lboost_system"
     AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <boost/filesystem/path.hpp>]],
 				    [[return 0;]])],
                [BOOST_FILESYSTEM_LIB=$ax_lib
